stanford_news.module

```diff
@@ -5,3 +5,21 @@
  */
 
 include_once 'stanford_news.features.inc';
+
+
+/**
+ * Implements hook_views_pre_render
+ */
+
+function stanford_news_views_pre_render(&$view) {
+
+  // Only pull in the css if it the news grid view is in use.
+  $valid = array(
+    'stanford_news_grid',
+  );
+
+  // Add the css for the news_grid view
+  if (in_array($view->name, $valid)) {
```

Comment:
There is only one view that this belongs to. You don't need the in_array option. Please change to $view->name == "stanford_news_grid"